packages/web/src/scss/theme/_form-fields.scss

```diff
@@ -34,12 +36,7 @@ $box-field-input-border-style: tokens.$border-style-100;
 $box-field-input-border-color-default: tokens.$border-primary-default;
 $box-field-input-border-color-disabled: tokens.$border-primary-disabled;
 $box-field-input-border-color-focus: tokens.$focus-default;
-$box-field-input-border-color-success: tokens.$emotion-success-default;
-$box-field-input-border-color-warning: tokens.$emotion-warning-default;
 
-// @deprecated Will be removed in the next major version.
```

Comment:
I probably owe an apology to @tomassychra, because if I remember correctly we were working on this together and I now realized we don't need to keep this level of variables. So, sorry Tomas :)